test/build.js

```diff
@@ -35,25 +32,25 @@ async function main () {
     keepNames: true,
     platform: 'browser',
     sourcemap: 'inline',
-    external: ['node:*'],
+    external: ['node:*', 'socket:*'],
     outdir: path.resolve(process.argv[2]),
     plugins: []
   }
 
-  if (os.platform() !== 'win32') {
-    params.external.push('socket:*')
-  } else {
-    params.plugins.push({
-      name: 'socket-runtime-import-path',
-      setup (build) {
-        build.onResolve({ filter: /^socket:.*$/ }, (args) => {
-          const basename = args.path.replace('socket:', '').replace(/.js$/, '') + '.js'
-          const filename = path.resolve(path.dirname(dirname), 'api', basename)
-          return { path: filename, external: true }
-        })
-      }
-    })
-  }
+  // if (os.platform() !== 'win32') {
```

Comment:
maybe a comment above explaining why this is commented out if we're going to leave it 🙏🏼